packages/mantine/src/theme/Theme.tsx

```diff
@@ -277,8 +277,9 @@ export const plasmaTheme: MantineThemeOverride = createTheme({
         }),
         ModalOverlay: Modal.Overlay.extend({
             defaultProps: {
-                color: color.primary.navy[9],
-                backgroundOpacity: 0.9,
+                color: PlasmaColors.indigo[8],
```

Comment:
Should we use something like `var(--mantine-color-primary-8)` ?

Reply:
UX specifically created a token for this indigo.8, it's the same in all hubs

<img width="1178" alt="image" src="https://github.com/user-attachments/assets/08832746-263b-4110-bca9-aeafd7f4235a" />